idd/Energy+.idd.in

```diff
@@ -1457,15 +1457,19 @@ WeatherProperty:SkyTemperature,
        \object-list RunPeriodsAndDesignDays
   A2,  \field Calculation Type
        \required-field
+       \note The field indicates that the sky temperature will be imported from external schedules or calculated by alternative methods other than default.
        \type choice
        \key ScheduleValue
        \key DifferenceScheduleDryBulbValue
        \key DifferenceScheduleDewPointValue
+       \key UseBruntModel
+       \key UseIdsoModel
+       \key UseBerdahlMartinModel
```

Comment:
ClarkAllen should be another choice here, and we can drop "Use" and "Model" from the key choices. So, this would be:
```
       \type choice
       \key ClarkAllen
       \key Brunt
       \key Idso
       \key BerdahlMartin (or MartinBerdahl???)
       \key ScheduleValue
       \key DifferenceScheduleDryBulbValue
       \key DifferenceScheduleDewPointValue
       \default ClarkAllen
```